crates/primitives/block/Cargo.toml

```diff
@@ -12,6 +12,11 @@ targets = ["x86_64-unknown-linux-gnu"]
 
```

Comment:
You must be the first person I review who does the "features thing" correctly! GG

Reply:
lfg!